python/lsst/sdm/schemas/efd_latiss.yaml

```diff
@@ -0,0 +1,2764 @@
+  - name: "at_hexapod_reported_position_z_mean"
+    "@id": "#exposure_efd.at_hexapod_reported_position_z_mean"
+    datatype: float
+    nullable: true
+    description: Mean auxiliary telescope hexapod reported position in z.
+  - name: "at_hexapod_reported_position_z_max"
+    "@id": "#exposure_efd.at_hexapod_reported_position_z_max"
+    datatype: float
+    nullable: true
+    description: Maximum auxiliary telescope hexapod reported position in z.
+  - name: "at_hexapod_reported_position_z_min"
+    "@id": "#exposure_efd.at_hexapod_reported_position_z_min"
+    datatype: float
+    nullable: true
+    description: Minimum auxiliary telescope hexapod reported position in z.
+  - name: "at_hexapod_reported_position_u_mean"
+    "@id": "#exposure_efd.at_hexapod_reported_position_u_mean"
+    datatype: float
+    nullable: true
+    description: Mean auxiliary telescope hexapod reported position in u.
+  - name: "at_hexapod_reported_position_u_max"
+    "@id": "#exposure_efd.at_hexapod_reported_position_u_max"
+    datatype: float
+    nullable: true
+    description: Maximum auxiliary telescope hexapod reported position in u.
+  - name: "at_hexapod_reported_position_u_min"
+    "@id": "#exposure_efd.at_hexapod_reported_position_u_min"
+    datatype: float
+    nullable: true
+    description: Minimum auxiliary telescope hexapod reported position in u.
+  - name: "at_hexapod_reported_position_v_mean"
+    "@id": "#exposure_efd.at_hexapod_reported_position_v_mean"
+    datatype: float
+    nullable: true
+    description: Mean auxiliary telescope hexapod reported position in v.
+  - name: "at_hexapod_reported_position_v_max"
+    "@id": "#exposure_efd.at_hexapod_reported_position_v_max"
+    datatype: float
+    nullable: true
+    description: Maximum uxiliary telescope hexapod reported position in v.
+  - name: "at_hexapod_reported_position_v_min"
+    "@id": "#exposure_efd.at_hexapod_reported_position_v_min"
+    datatype: float
+    nullable: true
+    description: Minimum auxiliary telescope hexapod reported position in v.
+  - name: "at_hexapod_reported_position_w_mean"
+    "@id": "#exposure_efd.at_hexapod_reported_position_w_mean"
+    datatype: float
+    nullable: true
+    description: Mean auxiliary telescope hexapod reported position in w.
+  - name: "at_hexapod_reported_position_w_max"
+    "@id": "#exposure_efd.at_hexapod_reported_position_w_max"
+    datatype: float
+    nullable: true
+    description: Maximum auxiliary telescope hexapod reported position in w.
+  - name: "at_hexapod_reported_position_w_min"
+    "@id": "#exposure_efd.at_hexapod_reported_position_w_min"
+    datatype: float
+    nullable: true
+    description: Minimum auxiliary telescope hexapod reported position in w.
+  - name: "at_salindex202_acceleration_x_mean"
+    "@id": "#exposure_efd.at_salindex202_acceleration_x_mean"
+    datatype: float
+    nullable: true
+    description: Mean acceleration in x direction.
```

Comment:
"Mean acceleration..." of what? (Similar issue with following columns.)

Reply:
x-axis direction. It was updated.